docs/en/ingest-management/index.asciidoc

```diff
@@ -81,6 +81,8 @@ include::elastic-agent/running-on-kubernetes-standalone.asciidoc[leveloffset=+3]
 
 include::elastic-agent/scaling-on-kubernetes.asciidoc[leveloffset=+3]
 
+include::elastic-agent/ingest-pipeline-in-kubernetes.asciidoc[leveloffset=+3]
```

Comment:
```suggestion
include::elastic-agent/ingest-pipeline-in-Kubernetes.asciidoc[leveloffset=+3]
```

Maybe it is sensitive to case letters and that is why it is failing

Reply:
Fixed it with lowercase